src/CppParser/Parser.cpp

```diff
@@ -4188,6 +4194,20 @@ Declaration* Parser::WalkDeclaration(const clang::Decl* D)
     return Decl;
 }
 
+int Parser::GetAlignAs(const clang::AlignedAttr* alignedAttr)
+{
+    using namespace clang;
+
+    if (alignedAttr->isAlignas())
+    {
+        if (!alignedAttr->isAlignmentErrorDependent() &&
```

Comment:
Merge the conditions.

Reply:
Originally it was a much longer method where that wasn't possible. I've also removed `using namespace clang` which is no longer needed.